metofficedatahub/multiple_files.py

```diff
@@ -55,12 +55,15 @@ class MetOfficeDataHub(BaseMetOfficeDataHub):
         except Exception as e:
             logger.debug(f"Could not make folder {folder_to_download} - {e}")
 
-    def download_all_files(self, order_ids: Optional[List[str]] = None):
-        """Download all files in the latest"""
```

Comment:
Would it be good to keep it optional? Perhaps its better to make sure we force it do use one (or mode) ids

Reply:
I've changed it from an optional because my understanding of how click works is that `order_ids` from the `run` is always not `None` - when running the program without specifying that parameter, the variable `order_ids` gets populated as `[]`. As such it seems safer to conform to that in the signature of `download_all_files`. 

Also, since we've changed the behaviour to make it so passing in no order ids downloads nothing, there is now no functional distinciton in the function output between `None` and `[]` so it seems to me more sensible to remove the ambiguity?

Happy to hear thoughts the other way though!

Reply:
yea sounds good, this seems sensible.